packages/cmf/README.md

```diff
@@ -67,21 +72,29 @@ Actions are [redux actions](http://redux.js.org/docs/basics/Actions.html).
 ### ComponentState Management
 
 Component state can be easily stored in cmf state, each are identified by their name and an unique key,
-so component state can be stored and reused later
+so component state can be stored and reused later.
+
+We give you the choice to use either:
+
+* React component state (this.state && this.setState)
```

Comment:
Should add warning that you should use CMF redux state before even thinking about local component state